Kitodo/src/test/java/test/SimpleLoginST.java

```diff
@@ -0,0 +1,411 @@
+
+        InternetAddress address = new InternetAddress(recipient);
+
+        ArrayList<InternetAddress> addressList = new ArrayList<>();
+        addressList.add(address);
+
+        // Create the attachment
+        EmailAttachment attachment = new EmailAttachment();
+        attachment.setPath(attachedFile.getAbsolutePath());
+        attachment.setDisposition(EmailAttachment.ATTACHMENT);
+        attachment.setDescription("SeleniumScreenShot");
+        attachment.setName("screenshot.png");
+
+        MultiPartEmail email = new MultiPartEmail();
+        email.setHostName("smtp.gmail.com");
+        email.setSmtpPort(465);
+        email.setAuthenticator(new DefaultAuthenticator(user, password));
+        email.setSSLOnConnect(true);
+        email.setFrom("Travis CI Screenshot <kitodo.dev@gmail.com>");
+        email.setSubject(subject);
+        email.setMsg(message);
+        email.setTo(addressList);
+        email.attach(attachment);
+
+        email.send();
+    }
+
+    private static File captureScreenShot(WebDriver driver) {
+
+        File src = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
+        File screenshotFile = new File(System.getProperty("user.dir") + "/target/Selenium/" + "screen.png");
+        try {
+            FileUtils.copyFile(src, screenshotFile);
+        } catch (IOException e) {
+            logger.error(e.getMessage());
+        }
+        return screenshotFile;
+    }
+
+    private static void setExecutePermission(File file) throws IOException {
+        Set<PosixFilePermission> perms = new HashSet<>();
+        perms.add(PosixFilePermission.OWNER_READ);
+        perms.add(PosixFilePermission.OWNER_WRITE);
+        perms.add(PosixFilePermission.OWNER_EXECUTE);
+
+        perms.add(PosixFilePermission.OTHERS_READ);
+        perms.add(PosixFilePermission.OTHERS_WRITE);
+        perms.add(PosixFilePermission.OTHERS_EXECUTE);
+
+        perms.add(PosixFilePermission.GROUP_READ);
+        perms.add(PosixFilePermission.GROUP_WRITE);
+        perms.add(PosixFilePermission.GROUP_EXECUTE);
+
+        Files.setPosixFilePermissions(file.toPath(), perms);
+    }
+
+    private static void extractZipFileToFolder(File zipFile, File destinationFolder) {
+        try {
+            ZipFile zip = new ZipFile(zipFile);
+
+            destinationFolder.mkdir();
+            Enumeration zipFileEntries = zip.entries();
+
+            while (zipFileEntries.hasMoreElements()) {
+                ZipEntry entry = (ZipEntry) zipFileEntries.nextElement();
+                String currentEntry = entry.getName();
+
+                File destFile = new File(destinationFolder, currentEntry);
+                File destinationParent = destFile.getParentFile();
+
+                destinationParent.mkdirs();
+
+                if (!entry.isDirectory()) {
+                    BufferedInputStream bufferedInputStream = new BufferedInputStream(zip.getInputStream(entry));
```

Comment:
Streams or any other closeable object (`bufferedInputStream`, `fileOutputStream` and  `bufferedOutputStream` in this block) should be handled by a [try-with-resources-statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html).